src/Microsoft.AspNetCore.Routing.Abstractions/EndpointMetadataCollection.cs

```diff
@@ -59,12 +99,27 @@ public EndpointMetadataCollection(params object[] items)
             }
         }
 
+        /// <summary>
+        /// Gets an <see cref="IEnumerator"/> of all metadata items.
+        /// </summary>
+        /// <returns>An <see cref="IEnumerator"/> of all metadata items.</returns>
         public Enumerator GetEnumerator() => new Enumerator(this);
 
+        /// <summary>
```

Comment:
I don't think you need docs on explicit implementations. The only way to access is to cast to the interface type.

Reply:
Ok well I did it so I'm not going to delort it 😆